general/array.hpp

```diff
@@ -174,6 +178,9 @@ class Array
    /// Append element 'el' to array, resize if necessary.
    inline int Append(const T & el);
 
+   /// STL-like push_back. Append element 'el' to array, resize if necessary.
+   void push_back(const T &el) { static_cast<void>(Append(el)); }
```

Comment:
Do you cast to void to avoid a warning about the ignored returned value? Shouldn't it only warn if `Append` is `nodiscard`?